core/hydromet_JSON_to_DSS.py

```diff
@@ -40,8 +44,11 @@ def main(forcing_dir: plib, outputs_dir: plib, bin_dir: plib, filename: str,
     files = []
     temp_files = ['DSSUTL.EXE', 'DSS_MAP.input']
     all_durations = []
-    for file in forcing_dir.glob('*.json'):
-        files.append(file)
+    if scaling is False:
+        for file in forcing_dir.glob('*.json'):
+            files.append(file)
+    if scaling is True:
+        files.append(pl.Path(forcing_dir)/pl.Path(jsonname))
```

Comment:
No need to have two pathlib objects, can just make the forcing_dir a pathlib object and then put /jsonname.

This can be an if else statement instead of two if statements.

Reply:
- Agreed. Removed second pathlib object 
- Agreed. Changed to an else statement